src/Components/Button/Button.jsx

```diff
@@ -0,0 +1,5 @@
+export default function Button({text, type}){
+    return(
+        <button className={`button button--${type}`}>{text}</button>

```

Comment:
I think that here is missing the style too, since you're implementing an isolated component, I suggest you add the visual style for this component and then put in the screenshot section on description a print about what the component looks like.